src/other_functions.jl

```diff
@@ -134,6 +129,13 @@ function abs{T<:Real}(a::Taylor1{TaylorN{T}})
         (abs(x) is not differentiable at x=0)."""))
     end
 end
+"""
+    abs(a::Union{Taylor1,TaylorN})
+
+Returns `a` if `constant_term(a) > 0` and `-a` if `constant_term(a) < 0`.
+Notice that `typeof(a) <: AbstractSeries`.
+"""
```

Comment:
You will still need to add `abs` after the (second) `"""`!

Reply:
Oh! I'm sorry! I didn't remember that!